backend/src/appsettings.development.json

```diff
@@ -9,5 +9,14 @@
   },
   "Database": {
     "ConnectionString": "Host=database; Port=5432; Database=xbase_development; User Id=postgres; Password=postgres; Maximum Pool Size=90;"
-  }
-}
+  },
+  //"Database": {
```

Comment:
Please remove these comments. They are invalid JSON (according to my understanding of JSON).